OpenSim/Tests/BodyDragExample/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Define test target and current test dir
+set(TEST_TARGET testBodyDragExample)
+
+function(CopyTestFiles SRC_DIR DST_DIR)
```

Comment:
Please rename to `OpenSimCopyTestFiles`; we seem to have this convention (see [here](https://github.com/opensim-org/opensim-core/blob/master/Bindings/Python/CMakeLists.txt#L17)).